cms/lib/xblock/tagging/tagging.py

```diff
@@ -43,7 +43,8 @@ def student_view_aside(self, block, context):  # pylint: disable=unused-argument
         Display the tag selector with specific categories and allowed values,
         depending on the context.
         """
-        if isinstance(block, CapaModule):
```

Comment:
@andy-armstrong
>For your issue, I think we need to work out why asides aren't rendering for author views.

It is because the content of the question is rendered in studio with a `student_view`. You can see this on my screenshot from the [comment above](https://github.com/edx/edx-platform/pull/12212#discussion_r60577795).
Asides that are created to be shown in `author_view` are shown only if the related Xblock is shown in the `author_view`. Particularly in this case problem is displayed in the `students_view`